transit_odp/organisation/querysets.py

```diff
@@ -1152,6 +1153,32 @@ def get_overall_data_catalogue(self):
     def filter_by_noc_and_line_name(self, noc, line_name):
         return self.filter(national_operator_code=noc, line_names__contains=[line_name])
 
+    def add_effective_stale_date_last_modified_date(self):
+        """Adds the Effective Stale Date: Last Modified Date of file + 1 year."""
+        return self.annotate(
+            effective_stale_date_last_modified_date=ExpressionWrapper(
+                F("modification_datetime") + timedelta(days=365),
+                output_field=DateField(),
+            )
+        )
+
+    def add_effective_stale_date_end_date(self):
+        """Adds the Effective Stale Date: Operating Period End Date - 42 days."""
+        return self.annotate(
+            effective_stale_date_end_date=ExpressionWrapper(
+                F("operating_period_end_date") - timedelta(days=42),
+                output_field=DateField(),
+            )
+        )
```

Comment:
Wouldn't these functions be quicker if you truncated to a Date (rather than DateTime)? You end up having to truncate later anyway.